INSTALL.md

```diff
@@ -84,7 +84,7 @@ Open **Terminal** and enter the following commands, depending on which distro yo
 To install the software required for **pokecrystal**:
 
 ```bash
-sudo apt-get install make gcc git
+sudo apt-get install make gcc git flex pkg-config
```

Comment:
You do not need `flex` or `pkg-config` do build pokecrystal. You may need them to build **rgbds**. Edit those instructions instead.